src/web/src/pages/opportunities/[[...query]]/index.tsx

```diff
@@ -575,36 +591,36 @@ const Opportunities: NextPageWithLayout<InputProps> = ({
 
       <div className="container z-10 max-w-7xl px-2 py-1 md:py-4">
         <div className="flex flex-col gap-2 pb-2 pt-8 text-white">
-          <h3 className="flex flex-grow flex-wrap items-center justify-center">
+          <h3 className="-mb-1 flex flex-grow flex-wrap items-center justify-center text-xl font-semibold">
             Find <span className="mx-2 text-orange">opportunities</span> to
             <span className="mx-2 text-orange">unlock</span> your future.
           </h3>
-          <h5 className="text-center">
+          <h6 className="text-center text-[14px] font-normal text-[#C3A2CD]">
```

Comment:
instead of hardcoding a color, you can add a new color to tailwind.config.ts e.g:

  colors: {
        "my-special-color" : "#C3A2CD"
        }
        
then use it as follows:
<div className="bg-my-special-color text-my-special-color" ...